API-Reference/API-Alphabetical.md

```diff
@@ -16,7 +16,7 @@ All functions and classes listed here are part of the `Kokkos::` namespace.
 ### Containers
 |Name |Library | Category | Description                  |
 |:---------|:--------|:-----------|:----------------------------|
-|[Bitset](Kokkos%3A%3ABitset) | [Containers](API-Containers) | View | A concurrent Bitset class. |
+|[Bitset](Bitset) | [Containers](API-Containers) | View | A concurrent Bitset class. |
```

Comment:
Did you forget to commit that file?
Also why did you not name it `[Kokkos::]Bitset` for consistency?

Reply:
The problem is we really aren't consistent.  We saw Unordered-Map.md in the top level directory, and started with that as the template for writing Bitset.

I don't mind changing it or moving the file.  Should I?

Reply:
Yeah change it back. Note Unorderedmap is not up to snuff in how it is written/structured.